src/Build/BuildCheck/Infrastructure/NullBuildCheckManager.cs

```diff
@@ -61,6 +61,10 @@ public void ProjectFirstEncountered(BuildCheckDataSource buildCheckDataSource, I
     }
 
     public void ProcessProjectEvaluationStarted(ICheckContext checkContext, string projectFullPath)
```

Comment:
Same as befor, do we need this extra function when it does not exist in the interface.

Reply:
Same as above